cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -298,6 +298,53 @@ resource localUserOpenAIIdentity 'Microsoft.Authorization/roleAssignments@2022-0
 }
 {{- end}}
 
+{{- if .AISearch}}
+module search 'br/public:avm/res/search/search-service:0.9.1' = {
+  name: 'ai-search'
+  params: {
+    name: '${abbrs.storageStorageAccounts}${resourceToken}'
```

Comment:
Do we need a new abbreviation for search here? I usually look up abbreviations here: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations

I had a tool available at one point available in a private [branch](https://github.com/Azure/azure-dev-pr/pull/1720/files#diff-a0d34c43dfde1ff99f6c4760e6ebd53380f0f184df59a2e1906fcd8df6ab9e4c) but haven't had time to find a way to get this integrated into mainline.

Reply:
Good catch! The abbreviation (`srch-`) is already in `abbreviations.json` so I just updated this.